debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -191,6 +191,15 @@
             .withDescription("Complete JDBC URL as an alternative to specifying hostname, port and database provided "
                     + "as a way to support alternative connection scenarios.");
 
+    public static final Field LOG_MINING_DML_PARSER = Field.create("log.mining.dml.parser")
```

Comment:
Let's make it an _internal_ option. Ideally, it shouldn't be needed, and we can refer users to it only in case they have issues with the new one.

Reply:
When you say _internal_, what specifically should I change here outside of not documenting this option?

Reply:
Ah; nvm, you likely mean `Field.createInternal`.

Reply:
Yes, exactly. Are you planning to make this change?